src/components/Floating-button/Floating-button.js

```diff
@@ -26,7 +27,11 @@ export default class FloatingButton extends React.Component {
 
   render() {
     return (
-      <div data-tooltip={this.props.tooltipMessage || null} id={this.state.id}>
+      <div
+        data-tooltip={this.props.tooltipMessage || null}
+        id={this.state.id}
+        className={`${MOUSE_CONTROL}`}
```

Comment:
Same everywhere

```suggestion
        className={MOUSE_CONTROL}
```

Reply:
No JC. I tried but the hierarchy is different.